cirq-google/requirements.txt

```diff
@@ -0,0 +1,2 @@
+google-api-core[grpc] >= 1.14.0, < 2.0.0dev
```

Comment:
Should `cirq-core` be part of these requirements?

Reply:
I don't think so, requirements.txt is for development. When you're developing cirq.google, you will have cirq-core checked out as well, so you should depend on that. 

However https://github.com/quantumlib/Cirq/pull/3957/commits/57687f989535e1c82a3258e0df0a1232bdecb112#diff-75c481d943df4a45c7599762865505aa8860d35baf87e3629911224dd37fa620R50 should capture the dependency of cirq-google on cirq-core and it doesn't, so I'll add it there!

Reply:
Oh ignore that - I was looking at the wrong file. 
We do capture the dependency in https://github.com/quantumlib/Cirq/commit/57687f989535e1c82a3258e0df0a1232bdecb112#diff-be82d3fb5dd590eca7a9d68afc211891571cfcb4816e05cc96355895f0b2b34aR52